essential-knowledge/cp-permissions.rst

```diff
@@ -0,0 +1,90 @@
+.. _knowledge-cp-permissions:
+
+Control panel permissions reference
+====================================
+
+The following table shows the different permission options for the different
+types of users with respect to addons, boilerplates and applications.
+
+
+Addons
+----------
+
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
+| User level    |  Create new | Manage addon | Manage addon  | Upload new    | Install/upgrade | Manage settings | 
```

Comment:
Where is the information about what is allowed/denied? We need some sort of indicator.

Reply:
I need more information on this.